scripts/common/prompts.sh

```diff
@@ -298,10 +250,6 @@ function prompt_airgap_preload_images() {
             logWarn "Remote load-images task prompt explicitly ignored"
         fi
     done < <(kubectl get nodes --no-headers)
-
-    if [ "$unattended_nodes_missing_images" = "1" ] ; then
-        bail "Preloading images required"
```

Comment:
i do not understand why this is changing

Reply:
This one was removed because `unattended_nodes_missing_images=1` scenario could only occurs if:

``` 
if ! prompts_can_prompt ; then
                unattended_nodes_missing_images=1
                continue
            fi
```

Therefore, we removed the `prompts_can_prompt ` so we do not need this one anymore. The `unattended_nodes_missing_images` is unused code now.